### PR TITLE
Load clipboard.min.js locally

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: INSTALL-XAMPP.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ XAMPP.
 
  1. Install [XAMPP](https://www.apachefriends.org/), e.g. in the folder `C:\xampp\`
  2. Clone malibu in the `htdocs` subfolder
- 3. Download [jQuery](https://code.jquery.com/jquery-3.2.1.min.js) and place it in `.\htdocs\malibu\isbn`
+ 3. Download [jQuery](https://code.jquery.com/jquery-3.2.1.min.js) and [clipboard.js 1.7.1](https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js) and place them in `.\htdocs\malibu\isbn`
```
Comment (stweil) [inline INSTALL-XAMPP.md]: Replace `clipboard.js` by `clipboard.min.js` here.

File: INSTALL.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ Some steps have to be performed after the server configuration and
 before the first start:
 
 1. Clone the repository or download all files
-2. Download [jQuery 3.2.1](https://code.jquery.com/jquery-3.2.1.min.js) into `isbn` directory
+2. Download [jQuery 3.2.1](https://code.jquery.com/jquery-3.2.1.min.js) and [clipboard.js 1.7.1](https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js) into `isbn` directory
```
Comment (stweil) [inline INSTALL.md]: Same as above.

File: INSTALL-XAMPP.md (hunk 1 above)
Comment (zuphilip) [reply INSTALL-XAMPP.md]: Well, the names of the libraries are jQuery and clipboard.js, and both of them should be downloaded in the minified version... Isn't it more unclear if we only write the minified version of clipboard but not jQuery?

Comment (stweil) [reply INSTALL-XAMPP.md]: You are right. I did not know that clipboard.js is the product name.

**Pull request merged**